samples/client/petstore/typescript-fetch/builds/default/apis/PetApi.ts

```diff
@@ -0,0 +1,396 @@
+// tslint:disable
+/**
+ * OpenAPI Petstore
+ * This is a sample server Petstore server. For this sample, you can use the api key `special-key` to test the authorization filters.
+ *
+ * OpenAPI spec version: 1.0.0
+ * 
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+
+import * as runtime from '../runtime';
+import {
+    ModelApiResponse,
+    ModelApiResponseFromJSON,
+    ModelApiResponseToJSON,
+    Pet,
+    PetFromJSON,
+    PetToJSON,
+} from '../models';
+
+export interface AddPetRequest {
+    pet: Pet;
+}
+
+export interface DeletePetRequest {
+    petId: number;
+    apiKey?: string;
+}
+
+export interface FindPetsByStatusRequest {
+    status: Array<'available' | 'pending' | 'sold'>;
+}
+
+export interface FindPetsByTagsRequest {
+    tags: Array<string>;
+}
+
+export interface GetPetByIdRequest {
+    petId: number;
+}
+
+export interface UpdatePetRequest {
+    pet: Pet;
+}
+
+export interface UpdatePetWithFormRequest {
+    petId: number;
+    name?: string;
+    status?: string;
+}
+
+export interface UploadFileRequest {
+    petId: number;
+    additionalMetadata?: string;
+    file?: Blob;
```

Comment:
support for blobs has been added. the pet store example doesn't include any blob responses, but binary body params and binary responses now work; using the following spec for testing: `modules/openapi-generator/src/test/resources/2_0/binaryDataTest.json`